scheduler-server/scheduling-dsl-compiler/src/libs/scheduler-edsl-fluent-api.ts

```diff
@@ -69,8 +69,8 @@ export class Goal {
   }
 }
 
-type StartTimingConstraint = { startsAt: SingletonTimingConstraint } | { startsWithin: RangeTimingConstraint }
```

Comment:
I recently rebased and in my version of scheduler-edsl-fluent-api.ts, I never had these lines. I'm not sure if that's a problem with this commit though or if it'll cause merging issues.

Reply:
I think you might be looking at a line that I changed twice in this PR - I first introduced `SingletonTimingConstraint`, and then expanded it to have a `SingletonTimingConstraintNoOperation`.